integration-test/build.gradle

```diff
@@ -5,9 +5,13 @@ buildscript {
     ext.serialization_version = mainLibVersion
 
     repositories {
-        mavenLocal()
         mavenCentral()
         maven { url "https://cache-redirector.jetbrains.com/maven.pkg.jetbrains.space/kotlin/p/kotlin/dev" }
+        mavenLocal() {
+            mavenContent {
```

Comment:
The local repository is only used to search for recently builded snapshot serialization artifacts?

Reply:
Yes